types/plotly.js/index.d.ts

```diff
@@ -1300,20 +1300,14 @@ export interface PlotData {
         | 'percent'
         | 'none';
     textposition:
```

Comment:
It looks like you narrowed the scope of the `textposition` to just the [Bar trace](https://github.com/plotly/plotly.js/blob/96c28159c06a3af3fe551175db89599894efbf82/src/traces/bar/attributes.js#L74), but other traces, like [Scatter](https://github.com/plotly/plotly.js/blob/96c28159c06a3af3fe551175db89599894efbf82/src/traces/scatter/attributes.js#L620) still use other values such as `top left`, `bottom right`, etc.

Recommend reverting this change.

Reply:
Reverted the changes